e2e/davinci-app/main.ts

```diff
@@ -0,0 +1,190 @@
+import './style.css';
+
+import { Config, FRUser, TokenManager } from '@forgerock/javascript-sdk';
+
+import davinci from '@forgerock/davinci-client';
+import usernameComponent from './components/text.ts';
+import passwordComponent from './components/password.ts';
+import submitButtonComponent from './components/submit-button.ts';
+import protect from './components/protect.ts';
+import flowLinkComponent from './components/flow-link.ts';
+import socialLoginButtonComponent from './components/social-login-button.ts';
+
+/**
+ * Import types for the DaVinci client
+ */
+import type { SingleValueCollector, ActionCollector } from '@forgerock/davinci-client/types.ts';
```

Comment:
i'm confused by this import, the `@forgerock/davinci-client` is used here but other places we are using relative imports?

Reply:
I think I just partially corrected some imports as I moved the code over to this repo. I'll make sure they're consistent.